templates/.vscode/extensions.json.tpl

```diff
@@ -14,6 +14,10 @@
     "laktak.hjson",
     {{- end }}
 
+    {{- range (stencil.GetModuleHook ".vscode/extensions") }}
```

Comment:
issue: I would generally prefer to have the identifier be decoupled from the file path and be more descriptive.
```suggestion
    {{- range (stencil.GetModuleHook "vscode/additional-extensions") }}
```